corehq/apps/users/templates/users/edit_commcare_user.html

```diff
@@ -136,16 +136,27 @@
         </div>
 
         <div class="tab-pane" id="groups">
+        {% if needs_to_downgrade_locations %}
+            <div>
+                <p>
+                    {% trans "This project no longer has access to the Organization feature." %}
+                    <a href="{% url "downgrade_locations" domain %}">
+                        {% trans "Click here to fix this." %}
```

Comment:
Similarly to above, if this is just going to be a big blank space with a link, how about giving them the controls to fix things here instead of requiring another page load?